testassets/InteropTestsWebsite/Resources.cs

```diff
@@ -0,0 +1,48 @@
+#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.IO;
+
+namespace InteropTestsWebsite
+{
+    public static class Resources
+    {
+        public static string CertDir = Path.Combine(GetSolutionDirectory(), "testassets/InteropTestsWebsite", "Certs");
+        public static string ServerPFXPath = Path.Combine(CertDir, "server1.pfx");
+
+        private static string GetSolutionDirectory()
```

Comment:
logic copied from examples for now, but I think a want to use a simple way.

Reply:
Yea, duplication is always undesired, not a big deal though.

Reply:
I'm not a fan of this way of referencing certs. Is there a reason why we have this pattern?

I'd prefer that we include the certs are content in the project, copy them to the output directory on build and reference them there.

Should change this both for the samples and the interop test site.

Reply:
I recall trying this when I set this up but I didn't get it to work. Copying to the output directory means the files will be in the `bin/Debug/netcoreapp3.0` folder. However, when you invoke `dotnet run`, the root directory is set to the project directory, which means you need to include `bin/Debug/netcoreapp3.0` in your path to the certificate file. We found that resolving the full path more robust (against Debug/Release for example). 

Is there any new csproj mechanisms that I'm not aware of that can make this work?

Reply:
Lets keep going with this for now Because We Know It Works, and clean it up later.

Reply:
https://github.com/grpc/grpc-dotnet/issues/94